src/functions/keycloak/update-user-attributes/handler.ts

```diff
@@ -60,14 +51,15 @@ const updateUserAttributeHandler: APIGatewayProxyHandler = async (event: APIGate
     }
     attributes["severa-user-id"] = [severaUser.guid];
 
-    await api.updateUserAttribute(id, attributes);
-
-    await severaApi.getUserByEmail(severaUser.email, attributes);
+    const updateResponse = await api.updateUserAttribute(id, attributes);
 
     return {
       statusCode: 200,
       body: JSON.stringify({
-        message: `Severa user updated with: ${JSON.stringify({ attributes })}, Keycloak attributes updated: ${JSON.stringify(attributes)}`
+        message: "Severa user and Keycloak attributes updated",
+        severaUser: severaUser,
+        updatedAttributes: attributes,
+        keycloakResponse: updateResponse
```

Comment:
This seems better to me, just remember we need to describe what is being returned in the spec